website/pages/docs/plugins/sources/aws/policies.md

```diff
@@ -0,0 +1,493 @@
+# Policies and Compliance Frameworks
```

Comment:
We should exclude this file via https://github.com/cloudquery/cloudquery/blob/812241697c644bdb1ae202bbadcb3baae456f788/.github/workflows/lint_markdown.yml#L19